asdf/asdf.py

```diff
@@ -111,7 +111,7 @@ def __init__(
             open during the lifetime of the tree. Setting to False causes
             all data arrays to be loaded up front, which means that they
             can be accessed even after the underlying file is closed.
-            Note: even if `lazy_load` is `False`, `copy_arrays` is still taken
+            Note: even if ``lazy_load`` is `False`, ``copy_arrays`` is still taken
```

Comment:
```suggestion
            Note: even if ``lazy_load`` is ``False``, ``copy_arrays`` is still taken
```

Reply:
Putting False in single ticks actually links it into the CPython docs.